tests/test_schema.py

```diff
@@ -5404,14 +5418,45 @@ def test_schema_migrations_equivalence_60(self):
                 required property name -> str;
                 index pg::spgist on (.name);
             };
-        """, r"""
+        """,
+                r"""
```

Comment:
Indent should line up.

ruff is not checking any whitespace errors, and it is a big problem.

Reply:
And `r"""` is indented 'correctly', which is one indentation more than `[`, which is one indentation more than `self._assert_migration_equivalence`.

But yea, looks weird.